apps/communications/contacts/js/views/details.js

```diff
@@ -82,6 +83,8 @@ contacts.Details = (function() {
 
     initPullEffect(cover);
 
+    LazyLoader.load('/contacts/js/nfc.js');
```

Comment:
If we do this lazy load here we could have potentially race conditions, later on you line 90 we are setting up the `handleDetailsBack` syncrhonously, which could potentially call `stopNFC` before it has been called.